third_party/Build.mk

```diff
@@ -69,7 +69,7 @@ third_party/localtests: cargo_version_check sandbox_setup build/elf2tab
 
 
 .PHONY: build/cargo-host/release/elf2tab
-build/cargo-host/release/elf2tab: cargo_version_check build/elf2tab
+build/cargo-host/release/elf2tab: cargo_version_check build/elf2tab sandbox_setup
```

Comment:
Shouldn't the `sandbox_setup` be before the `build/elf2tab`?

Reply:
That bugfix is actually incorrect -- `build/cargo-host/release/elf2tab` does not need `sandbox_setup`. I put it last because it was last alphabetically, but I removed the commit that added it because that commit is incorrect.

I'm not sure why things didn't work for tjshep, but they're unrelated to this PR.

Reply:
Actually, I don't know what I was thinking when I wrote the previous comment. Yes, it needs `sandbox_setup`.

I re-added it, and this time I put the dependencies into alphabetical order.